HOT Movies/HOT Movies/Scenes/Movie Deltail Screen/MovieDetailsViewModel.swift

```diff
@@ -34,13 +34,13 @@ struct MovieDetailsViewModel {
                 return self.useCase.getMovieDetails(movieId: self.movie.idMovie)
                     .asDriverOnErrorJustComplete()
             }
-            .map {
-                [.detail(items: [
-                        .info(model: $0),
-                        .description(model: self.movie.overview),
-                        .castAndCrew(model: $0.credits),
-                        .similar(model: $0.similar.results)
-                ])]
+            .map { movieDetails -> [DetailsSectionModel] in
+                return [.detail(items: [
+                                .info(model: movieDetails),
+                                .description(model: movieDetails.overview),
+                                .castAndCrew(model: movieDetails.credits),
+                                .similar(model: movieDetails.similar.results)
+                        ])]
```

Comment:
Em viết như trên nó không work anh ạ. Nó lỗi Type Any has not detail . info ..v..v. nó không indent được kiểu trả về [DetailsSectionModel] ạ.

Reply:
thế giữ như này được nhé

Reply:
Vâng ạ.